pennylane/ops/op_math/pow_class.py

```diff
@@ -253,17 +252,15 @@ def generator(self):
         return self.z * self.base.generator()
 
     def simplify(self) -> Union["Pow", Identity]:
-        if self.z == 0:
-            return (
-                qml.prod(*(qml.Identity(w) for w in self.wires))
-                if len(self.wires) > 1
-                else qml.Identity(self.wires[0])
-            )
-        if isinstance(self.base, Controlled):  # Pow(Controlled(base)) = Controlled(Pow(base))
-            return Controlled(
-                base=Pow(self.base.base.simplify(), z=self.z),
-                control_wires=self.base.control_wires,
-                control_values=self.base.control_values,
-                work_wires=self.base.work_wires,
-            )
-        return Pow(base=self.base.simplify(), z=self.z)
+        try:
+            ops = self.base.pow(z=self.z)
+            if not ops:
+                return (
+                    qml.prod(*(qml.Identity(w) for w in self.wires))
+                    if len(self.wires) > 1
+                    else qml.Identity(self.wires[0])
+                )
+            op = qml.prod(*ops) if len(ops) > 1 else ops[0]
```

Comment:
Does this recover the behaviour for the controlled Op?

Reply:
`Controlled` has a `pow` method defined.

Reply:
Yes!